src/tree/values/FormattingElement.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Represents formatting elements (e.g. <strong>, <a>, <em>), except text, within an HTML-text.
```

Comment:
As discussed, this is supposed to represent source independent data. Please look for the other places too.

```suggestion
 * Represents formatting elements (e.g. <strong>, <a>, <em>), except text, within an text.
```

Reply:
Changed the docs so they are source agnostic instead of focused on HTML.